packages/widgetbook_core/lib/src/icons/component_icon.dart

```diff
@@ -0,0 +1,35 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+
+class ComponentIcon extends StatelessWidget {
+  const ComponentIcon({
+    super.key,
+    this.size = 12,
+  });
+
+  final double size;
+
+  @override
+  Widget build(BuildContext context) {
+    return Transform.rotate(
+      angle: 45 * pi / 180,
+      child: SizedBox(
+        width: size,
+        height: size,
+        child: GridView.count(
+          crossAxisCount: 2,
+          padding: EdgeInsets.zero,
```

Comment:
Nice, so this is not an icon but a "vector" component

Reply:
Yup 😅